### PR TITLE
Add blog post for Managing Beam dependencies in Java

### PR DESCRIPTION
**Pull request by bvolpato**
We have published BOMs a couple of releases back (thanks @kileys!), which makes it much easier to manage dependencies, but we didn't talk about it anywhere.

This may help raising some awareness, and can be linked whenever we have to explain the proper way of importing Beam (which comes every once in a while).

### CODE REVIEW

**Comment by AnandInguva** [comment]
@bvolpato I think its would better to tag someone with Beam Java knowledge

**Comment by AnandInguva** [comment]
stop reviewer notifications

**Comment by bvolpato** [comment]
R: @chamikaramj / @byronellis (who suggested a blog post)

**Comment by bvolpato** [comment]
Maybe @Abacn or @kennknowles will have some ideas / interest in checking this post too?

**Review by liferoad — APPROVED** [review]

File: website/www/site/content/en/blog/managing-beam-dependencies-in-java.md
Comment (Abacn) [inline website/www/site/content/en/blog/managing-beam-dependencies-in-java.md:49]:
```suggestion
- `beam-sdks-java-bom`, which manages what dependencies of Beam will be used, so
```

File: website/www/site/content/en/blog/managing-beam-dependencies-in-java.md
Comment (Abacn) [inline website/www/site/content/en/blog/managing-beam-dependencies-in-java.md:52]:
```suggestion
  which manages Beam, along with GCP client and third-party dependencies.
```

**Review by Abacn — APPROVED** [review]
Thanks @bvolpato for sharing! Just had a couple of minor comments

**Comment by bvolpato** [comment]
Accepted suggestions, thanks @Abacn

File: website/www/site/content/en/blog/managing-beam-dependencies-in-java.md
Comment (chamikaramj) [inline website/www/site/content/en/blog/managing-beam-dependencies-in-java.md:112]: Nit: should these links also be generic ? (without a specific Beam version) ?

File: website/www/site/content/en/blog/managing-beam-dependencies-in-java.md
Comment (chamikaramj) [inline website/www/site/content/en/blog/managing-beam-dependencies-in-java.md:120]: Add https://beam.apache.org/community/contact-us/  as a link for feedback ?

**Review by chamikaramj — COMMENTED** [review]
Thanks. LGTM other than a couple of nits.

File: website/www/site/content/en/blog/managing-beam-dependencies-in-java.md
Comment (bvolpato) [reply website/www/site/content/en/blog/managing-beam-dependencies-in-java.md]: Done, thanks

File: website/www/site/content/en/blog/managing-beam-dependencies-in-java.md
Comment (bvolpato) [reply website/www/site/content/en/blog/managing-beam-dependencies-in-java.md]: Added! Thanks

**Review by chamikaramj — APPROVED** [review]

**Comment by bvolpato** [comment]
Please merge this when someone can (not a committer).

**Pull request merged**
